moveos/moveos-stdlib/src/natives/moveos_stdlib/raw_table/mod.rs

```diff
@@ -8,29 +8,51 @@ use move_binary_format::errors::{Location, PartialVMError, PartialVMResult, VMRe
 use move_core_types::{
```

Comment:
Can we rename `raw_table` to `object_runtime`?

Reply:
TODO